tests/bcr/proto/foo.proto

```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
+
+package tests.bcr.proto;
+option go_package = "example.com/foo_proto";
```

Comment:
Is it possible to import from the standard library? Something like "google/protobuf/empty.proto" just to prove it can be done?

Reply:
@achew22 Yes, added.